packages/atlas-react/src/components/BasicButton/BasicButton.scss

```diff
@@ -1,7 +1,159 @@
+  --#{$namespace}-button-padding: #{spacer(mini)} #{spacer(small)};
+  --#{$namespace}-button-height: 36px;
+  --#{$namespace}-button-font-size: var(--a-size-font-body-14);
+  --#{$namespace}-button-width: auto;
+  --#{$namespace}-button-leading-icon-margin: 0 #{spacer(mini)} 0 0;
+  --#{$namespace}-button-trailing-icon-margin: 0 0 0 #{spacer(mini)};
+}
+
+.BasicButton.BasicButton--size-full-width  {
+  --#{$namespace}-button-padding: #{spacer(mini)} #{spacer(small)};
+  --#{$namespace}-button-height: 36px;
+  --#{$namespace}-button-font-size: var(--a-size-font-body-14);
+  --#{$namespace}-button-width: 100%;
+  --#{$namespace}-button-leading-icon-margin: 0 #{spacer(mini)} 0 0;
+  --#{$namespace}-button-trailing-icon-margin: 0 0 0 #{spacer(mini)};
+}
+
+.BasicButton.BasicButton--size-small  {
+  --#{$namespace}-button-padding: #{spacer(micro)} calc(#{spacer(micro)} + #{spacer(mini)});
+  --#{$namespace}-button-height: 28px;
+  --#{$namespace}-button-font-size: var(--a-size-font-body-14);
+  --#{$namespace}-button-width: auto;
+  --#{$namespace}-button-leading-icon-margin: 0 #{spacer(micro)} 0 0;
+  --#{$namespace}-button-trailing-icon-margin: 0 0 0 #{spacer(micro)};
+}
+
+.BasicButton.BasicButton--size-mini  {
+  --#{$namespace}-button-padding: #{spacer(micro)} #{spacer(mini)};
+  --#{$namespace}-button-height: 26px;
+  --#{$namespace}-button-font-size: var(--a-size-font-body-12);
+  --#{$namespace}-button-width: auto;
+  --#{$namespace}-button-leading-icon-margin: 0 #{spacer(micro)} 0 0;
+  --#{$namespace}-button-trailing-icon-margin: 0 0 0 #{spacer(micro)};
+}
+
+.BasicButton.BasicButton--align-left {
+  --#{$namespace}-button-align: flex-start;
+}
+
+.BasicButton.BasicButton--align-right {
+  --#{$namespace}-button-align: flex-end;
+}
+
+.BasicButton.BasicButton--align-center {
+  --#{$namespace}-button-align: center;
+}
+
+.BasicButton {
   & + & {
     margin-left: spacer(small);
   }
 }
+
+.BasicButton[disabled] {
+  pointer-events: none;
+}
+
+.BasicButton {
+  @include fontFamily('body');
+  padding: var(--#{$namespace}-button-padding);
+  background-color: var(--#{$namespace}-button-background-color);
+  border-radius: 4px;
+  border: none;
+  color: var(--#{$namespace}-button-color);
+  font-size: var(--#{$namespace}-button-font-size);
+  line-height: 1;
+  display: inline-flex;
+  align-items: center;
+  justify-content: var(--#{$namespace}-button-align);
+  height: var(--#{$namespace}-button-height);
+  cursor: pointer;
+  width: var(--#{$namespace}-button-width);
+
+  &:focus {
+    outline: none;
+    box-shadow: 0px 0px 0px 2px var(--#{$namespace}-button-background-color-focus-shadow);
+    background-color: var(--#{$namespace}-button-background-color-focus);
+  }
+
+  &:active {
+    background-color: var(--#{$namespace}-button-background-color-active);
+  }
+
+  &:disabled {
+    opacity: 0.5;
+  }
+
+  &:hover {
+    background-color: var(--#{$namespace}-button-background-color-hover);
+  }
+
+  .trailingIcon {
+    i {
```

Comment:
what happens if we just use `←`, for example?

Reply:
ah, perhaps I should have just converted this to .trailingIcon > *